app/components/Navbar/fixture.ts

```diff
@@ -30,41 +39,45 @@ export const NAVBAR_ITEMS: NavbarItem[] = [
     title: 'Ecosystem',
     link: '/ecosystem'
   },
-  {
-    dropdown: false,
-    title: 'Docs',
-    link: 'https://docs.etherlink.com/'
-  },
-  {
-    dropdown: false,
-    title: 'Faucet',
-    link: 'https://faucet.etherlink.com/'
-  },
-  {
-    dropdown: false,
-    title: 'Status',
-    link: 'https://status.etherlink.com/'
-  },
   {
     dropdown: true,
-    title: 'Resources',
+    title: 'Build',
     items: [
       {
-        name: 'Testnet explorer',
-        link: 'https://testnet-explorer.etherlink.com/'
+        name: 'Docs',
+        link: 'https://docs.etherlink.com/'
       },
       {
-        name: 'Mainnet beta explorer',
-        link: 'https://explorer.etherlink.com/'
+        name: 'Use your wallet',
+        link: 'https://docs.etherlink.com/get-started/using-your-wallet/'
       },
       {
-        name: 'Blog',
-        link: 'https://medium.com/@etherlink'
+        name: 'Tezos Bridge',
+        link: 'https://testnet.bridge.etherlink.com/',
+        subItems: [
+          {
+            name: 'Tezos Testnet',
+            link: 'https://testnet.bridge.etherlink.com/testnet'
+          },
+          {
+            name: 'Tezos Mainnet',
+            link: 'https://bridge.etherlink.com/mainnet'
```

Comment:
these 2 links are 404, can you double check?

Reply:
Amended